examples/basic_scan_async/main.go

```diff
@@ -17,37 +17,34 @@ func main() {
 	// with a 5 minute timeout.
 	s, err := nmap.NewScanner(
 		nmap.WithTargets("google.com", "facebook.com", "youtube.com"),
-		nmap.WithPorts("80,443,843"),
+		nmap.WithPorts("80,443,843,1-100"),
+		nmap.WithStatsEvery("1s"),
```

Comment:
WDYT of changing my implementation of `WithStatsEvery` to take a `time.Duration` instead of a string, and to then call `.String()` on the duration to get the same result? This would make it more idiomatic and user friendly, I think :) Can be done in another PR though!